src/cargo/core/workspace.rs

```diff
@@ -1006,10 +1006,12 @@ impl<'cfg> Workspace<'cfg> {
                         self.config.shell().warn(
                             format_args!("\
                         some crates are on edition {edition} which defaults to `resolver = \"{resolver}\"`,\n\
-                     \x20   but a virtual workspace defaults to `resolver = \"1\"`\n\
+                     \x20   but virtual workspaces default to `resolver = \"1\"`\n\
                      \x20   specify the desired resolver version explicitly in the workspace root's manifest\
```

Comment:
Did you mean to delete this line?  Seems a bit redundant with th new notes

Reply:
I was thinking that it made sense to have the instruction on what must be done to get rid of the warning, and then the options for how to do it, but if I copy the "in the workspace root's manifest" onto the notes too I think it reads ok (the duplication isn't great, but I think acceptable).